src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -453,6 +453,14 @@ doclet.usage.linkoffline.parameters=\
 doclet.usage.linkoffline.description=\
     Link to docs at <url1> using package list at <url2>
 
+# L10N: do not localize the option parameters: warn info
+doclet.usage.link-modularity-mismatch.parameters=\
+    (warn|info)
+doclet.usage.link-modularity-mismatch.description=\
+    Report external documentation with wrong modularity as either\n\
+    warning or informational message. The default behaviour is to\n\
```

Comment:
suggest:

> Report external documentation with wrong modularity with either\n\
     a warning or informational message.

(change "as" to "with"; add "a" before "warning")